File: ABC/A/260/A.cpp
#include<iostream>
#include<map>
using namespace std;

int main(){
    string s;
    cin >> s;
    map<char, int> m;
    for(auto c: s) m[c] = 0;
    for(auto c: s) m.at(c)++;
    string ans;
    for(auto iter: m){
        if(iter.second == 1){
            ans += iter.first;
            break;
        }
    }   
    if(ans.length() == 1) cout << ans << endl;
    else cout << -1 << endl;
}

File: ABC/A/260/A.py
s = input()
d = {}
for c in s: d[c] = 0
for c in s: d[c] += 1
ans = ""
for key in d:
    if d[key] == 1:
        ans = key
        break
print(ans if len(ans)!=0 else -1)